uninstall.sh

```diff
@@ -459,7 +459,12 @@ then
   fi
 fi
 
-dir_children "${HOMEBREW_REPOSITORY}" "${HOMEBREW_PREFIX}" |
+residual_dirs=("${HOMEREW_REPOSITORY}")
```

Comment:
Noticed there's a typo in this variable name:

```suggestion
residual_dirs=("${HOMEBREW_REPOSITORY}")
```

Reply:
<facepalm>